model_details.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Add this file to the .gitignore perhaps, just like the .europy folder is.

Reply:
can do, figured it would be easier for us to test while in development.

Reply:
I moved this as an example into test folder.